owncloudApp/src/main/java/com/owncloud/android/shares/ui/fragment/PublicShareDialogFragment.kt

```diff
@@ -146,33 +155,43 @@ class PublicShareDialogFragment : DialogFragment() {
      * Get expiration date imposed by the server, if any
      */
     private val imposedExpirationDate: Long
-        get() = if (mCapabilities != null && mCapabilities!!.filesSharingPublicExpireDateEnforced.isTrue) {
+        get() = if (capabilities != null && capabilities!!.filesSharingPublicExpireDateEnforced.isTrue) {
 
             DateUtils.addDaysToDate(
                 Date(),
-                mCapabilities!!.filesSharingPublicExpireDateDays
+                capabilities!!.filesSharingPublicExpireDateDays
```

Comment:
some lines above you are checking capabilities can be null, but here it won't. Are you sure?

Reply:
I cannot remove !! because of 

![Screenshot 2019-03-28 at 12 44 57](https://user-images.githubusercontent.com/15261363/55155314-531e2580-5157-11e9-9ee3-04a945ea9c53.png)